server/form.py

```diff
@@ -0,0 +1,96 @@
+import re
+
+from flask_wtf import FlaskForm
+from wtforms import BooleanField, SelectMultipleField, StringField, SubmitField, \
+    TextAreaField, widgets
+from wtforms.validators import Email, InputRequired, URL
+
+from server.utils.url import offering_regex, exam_regex
+
+
+class ExamForm(FlaskForm):
+    display_name = StringField('display_name', [InputRequired()], render_kw={
+                               "placeholder": "Midterm 1"})
+    name = StringField('name', [InputRequired()], render_kw={"placeholder": "midterm1"})
+    submit = SubmitField('create')
+
+    def validate_name(form, field):
+        pattern = '^{}$'.format(exam_regex)
```

Comment:
Is this a placeholder?

Reply:
It is not. 

Note the exam_regex imported from another file.